### PR TITLE
Avoid tooltip overlap with vLabel

### PR DESCRIPTION
**Pull request by maxmcd**

### CODE REVIEW

File: src/js/Rickshaw.Graph.HoverDetail.js [hunk 1]
```diff
@@ -180,14 +180,26 @@ Rickshaw.Graph.HoverDetail = Rickshaw.Class.create({
...
 
 		this.element.appendChild(item);
 
 		var dot = document.createElement('div');
 
 		dot.className = 'dot';
-		dot.style.top = item.style.top;
+		// the dot should be in the right position regardless of altering 
+		// the tooltip location
+		dot.style.top = topInt + "px";
```
Comment (nrlakin) [inline src/js/Rickshaw.Graph.HoverDetail.js]:
```suggestion
		dot.style.top = `${ topInt }px`
```
integer/string concatenation is gross

Comment (maxmcd) [reply src/js/Rickshaw.Graph.HoverDetail.js]:
```
src/js/Rickshaw.Graph.HoverDetail.js: line 202, col 25, 'template literal syntax' is only available in ES6 (use 'esversion: 6').
```
No 🎲

Comment (nrlakin) [reply src/js/Rickshaw.Graph.HoverDetail.js]: Ha!

**Comment by maxmcd** [comment]
@evanemolo done 👍 

I thought I added you as a reviewer, sorry that was an unintentional oversight.

**Pull request merged**